<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JGPA</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="7">
     <widget class="QTableWidget" name="mark_view">
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="credit_label">
      <property name="text">
       <string>Enter credit:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="credit_edit"/>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="curr_credit_label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="4" colspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>51</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="6">
     <widget class="QLabel" name="curr_mark_label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="mark_label">
      <property name="text">
       <string>Enter mark: </string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="mark_edit"/>
    </item>
    <item row="2" column="3" colspan="3">
     <widget class="QComboBox" name="marks_combo_box">
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4.20 (A+)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4.00 (A)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3.75 (A-)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3.50 (B+)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3.25 (B)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3.00 (B-)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2.75 (C+)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2.50 (C)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2.25 (C-)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2.00 (D+)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1.75 (D)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1.50 (D-)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0.50 (F)</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="6">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>137</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="mark_add_button">
      <property name="text">
       <string>Add mark</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="3">
     <widget class="QPushButton" name="mark_delete_button">
      <property name="text">
       <string>Delete mark</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5" colspan="2">
     <widget class="QPushButton" name="gpa_button">
      <property name="text">
       <string>Calculate GPA</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>412</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProfiles">
    <property name="title">
     <string>Profiles</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
   </widget>
   <addaction name="menuProfiles"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
